specifyweb/frontend/js_src/lib/components/Molecules/ResourceLink.tsx

```diff
@@ -25,6 +25,7 @@ export function ResourceLink<COMPONENT extends typeof Link['Icon']>({
   props: rawProps,
   resourceView,
   children,
+  autoClose = false,
```

Comment:
Should this be `autoClose=true` since the previous default behavior was dialog closing on resource changing? Now, the default behavior is dialog not closing on resource changing.

Reply:
oh, good catch!
@CarolineDenis could you open a small fixup pr for this?